kythe/rust/extractor/src/bin/extractor.rs

```diff
@@ -67,13 +67,29 @@ fn main() -> Result<()> {
         }
     }
 
+    // Grab the build target's output path
+    let build_output_path = spawn_info
+        .get_output_file()
+        .get(0)
+        .ok_or_else(|| anyhow!("Failed to get output file from spawn info"))?;
+
+    // Get the name of the output file (without extension)
+    let output_file_name = PathBuf::from(build_output_path)
+        .with_extension("")
+        .file_name()
+        .unwrap()
```

Comment:
Shouldn't this also be an `or_else` or equivalent rather than panicking on error?

Reply:
The `file_name` only returns `None` if the path ends with `..`, but I have gone ahead and added the `ok_or_else`